### DISCUSSION TITLE
can't cut and convert to MP3

### DISCUSSION THREAD

**Discussion by superryan14** [mifi/lossless-cut#603] [Q&A]
I'm probably missing something really simple as i'm not the greatest with this stuff but I wanted to see if it was a problem or just something that's on my end or the program not being able to convert the audio from a video file to an MP3 but I can't seem to cut a video file and export it as an MP3 with just audio. I have FFMPEG in my path and converting via bat files works fine.

If this isn't supported then I understand! Just wanted to make sure I wasn't doing something wrong. Thanks!

**Comment by Shakil-Shahadat** [comment]
Audio conversion is not supported by Lossless-cut. But you should be able to extract the audio in its original format.

**Comment by soloturn** [comment]
additional links in for opus, e-ac3, mp3:
* https://www.reddit.com/r/jellyfin/comments/ivgdyc/opus_vs_eac3/
* https://hydrogenaud.io/index.php?topic=113020.0

**Comment by mifi** [comment]
@superryan14 you can "export each track as separate files", then the audio track will be extract the audio in its original codec. If it's an mp3 inside the video file, then it will be an mp3. However like @Shakil-Shahadat  says you cannot convert codec as that is a lossy operation. use something like handbrake for that. @soloturn  I'm not sure what we need that for?
